add pangenomics test data for the #pangenome pipeline (#406)
nf-core/test-datasets: README.md

      * fastq
        * 'test.fastq.gz' synthetic raw long-read sequencing reads of the genome of the mammalian-gut-residing _Bacteroides fragilis_ YCH46 bacterium (NC_006347). Originally generated for the [MAG pipeline test dataset](https://github.com/nf-core/test-datasets/tree/mag).

### pangenomics

* homo_sapines
  * pangenome

>     * 'pangenome.fa': A FASTA file which contains several related genomes for the input of `wfmash`.

Reviewer: ```suggestion
    * 'pangenome.fa': A FASTA file which contains several related genomes.
```
i guess other tools could use this too, right?

Author: I guess some of them. That depends on their downstream way of handling things.

>     * 'pangenome.fa.gz': A GZIP compressed FASTA file which contains several related genomes for the input of `wfmash`.

Reviewer: ```suggestion
    * 'pangenome.fa.gz': A GZIP compressed FASTA file which contains several related genomes.
```

>     * 'pangenome.paf': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. 

Reviewer 2: ```suggestion
    * 'pangenome.paf': A PAF file which contains the all versus all pairwise alignments of related genomes. 
```

Author: Whoopsi, thanks!

>     * 'pangenome.paf': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. Ideally, it was generated by `wfmash`. It is consumed by `seqwish`.

Reviewer: ```suggestion
    * 'pangenome.paf': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. Ideally, it was generated by `wfmash`. It is consumed by `seqwish`.
```
Don't quite understand this sentence. This PAF file isgenerated by wfmash? I would remove the "how to generate" here and add it to the other readme to keep this file as overview

Author: Yes, such a PAF file was created by `wfmash`. 
Sorry, I should add this to which README.md?

Author: FOund it!

Author: I am not sure if `seqwish` could work without having the all versus all pairwise alignments. I would suggest leaving the text here.

>     * 'pangenome.paf.gz': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. Ideally, it was generated by `wfmash`. It is consumed by `seqwish`.

Reviewer: see above

>     * 'pangenome.seqwish.gfa': A GFA file which contains the pangenome graph induced by seqwish encoded in the variation graph model. It is consumed by `smoothxg`, `odgi sort`, `odgi build`, and `odgi stats`.

Reviewer: see above

>     * 'pangenome.smoothxg.gfa': A GFA file which contains the `smoothxg` smoothed pangenome graph. It is consumed by `odgi sort`, and `GFAffix`.

Reviewer: ```suggestion
    * 'pangenome.smoothxg.gfa': A GFA file which contains the `smoothxg` smoothed pangenome graph. 
```

>     * 'pangenome.gfaffix.gfa': A GFA file which was normalized with `gfaffix`. It is consumed by `vg deconstruct`, `odgi layout`, `odgi draw`, and `odgi viz`.

Reviewer: ```suggestion
    * 'pangenome.gfaffix.gfa': A GFA file which was normalized with `gfaffix`. 
```

  * odgi

>     * 'pangenome.og': A variation graph encoded in the binary ODGI format. It is consumed by `odgi view`.

Reviewer: ```suggestion
    * 'pangenome.og': A variation graph encoded in the binary ODGI format.
```

Author: This is a binary file which can only be read by `odgi`. I would leave it as it is.

>     * 'pangenome.lay': A binary file which holds the 2D graph layout produced by `odgi layout`. Input for `odgi draw`.

Reviewer: ```suggestion
    * 'pangenome.lay': A binary file which holds the 2D graph layout produced by `odgi layout`.
```

Author: This is a binary file specifically designed to be only readable by `odgi layout`. I would leave this here.
